packages/anoncreds/src/protocols/proofs/v1/__tests__/v1-connectionless-proofs.e2e.test.ts

```diff
@@ -134,6 +134,7 @@ describe('V1 Proofs - Connectionless - Indy', () => {
 
     await aliceAgent.proofs.acceptRequest({
       proofRecordId: aliceProofExchangeRecord.id,
+      hasReturnRoute: returnRoute,
```

Comment:
We're only passing `hasReturnRoute` when we're calling `acceptRequest`. What's the scenario for adding this when calling `acceptPresentation`?

Reply:
I deliberately only put it in acceptRequest as a quick fix for the use case causing our problem (which was when we were accepting a request and then timing out because no ack was received from the verifier).

My understanding (from James) was that there would be a fuller solution.

That said, no reason why we can't add it to acceptPresentation as well.

Reply:
No reason to add it I think, was just asking because you added it: https://github.com/NB-MikeRichardson/aries-framework-javascript/blob/feat/return-route/packages/core/src/modules/proofs/ProofsApiOptions.ts#L121